java/arcs/core/data/HandleMode.kt

```diff
@@ -18,14 +18,26 @@ enum class HandleMode {
     Read,
     /** [Handle] is write only. */
     Write,
+    /** [Handle] is query only. */
+    Query,
```

Comment:
So this means fetchAll() is unavailable, right?

Reply:
If a handle is query only, yes, I would not expect fetchAll to be available and querying would be the only method of data access.